### PR TITLE
feedback and steering

### PR DESCRIPTION
**Pull request by xfde**

### CODE REVIEW

File: docs/guides/refining-solutions/steering.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+## Matching
+
+* `match strategy`: 
+    * schema – use only attribute names for matching. 
+    * instance – use only instance evidence when there are instances in the data context, and when not use schema.
+    * schema + instances – combine the evidence from schema and instance matching into a single score.
+
+    > We recommend schema + instances, as it takes into account all available evidence.
+
+* `match threshold`: Value in the range 0 to 1. Discards all matches with a score below the threshold.
```
Comment (AspieCoder1) [inline docs/guides/refining-solutions/steering.md]: Make bold not inline code

**Review by AspieCoder1 — CHANGES_REQUESTED** [review]
Please can you change the inline code to bold? Only use inline code for actual code not for emphasis.

**Review by xfde — COMMENTED** [review]
Changed parameters text to bold

**Review by AspieCoder1 — APPROVED** [review]
Looks good. Thank you changing the stuff to bold

**Pull request merged**